requirements-pants.txt

```diff
@@ -45,7 +46,8 @@ prettytable
 # For st2client: prompt-toolkit v2+ does not have prompt_toolkit.token.Token
 prompt-toolkit<2
 psutil
-pymongo
+# pymongo 3.13 has backports of APIs from pymongo 4 to help w/ migration
+pymongo>=3.11.0,<3.13.0
```

Comment:
So 3.13 comes already with breaking changes? I'll study their changelog to get a better understanding. Besides that the PR looks good!

Reply:
It at least has a bunch of deprecations that are likely to cause (many?) extra warnings in the logs.